packages/create-cloudflare/src/index.d.ts

```diff
@@ -0,0 +1,7 @@
+export interface Argv {
+	init?: boolean;
+	force?: boolean;
+	debug?: boolean;
+}
```

Comment:
Feels like this file ought to be generated at build/publish time and not checked in?

Reply:
And probably move the `Argv` type to index.ts or utils.ts?

Reply:
package removed.